packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -276,20 +273,48 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
     );
   }
 
+  private loadingAnimationContent() {
+    const {isLoading, isStreaming} = this.generatedAnswerState;
+
+    if (isLoading || this.shouldBeHidden) {
+      return (
+        <div class="placeholder-wrapper relative overflow-hidden">
+          <GeneratedAnswerPlaceholder />
+        </div>
+      );
+    } else if (isStreaming) {
+      return (
+        <div class="placeholder-wrapper relative overflow-hidden">
+          <div class="z-1 bg-white relative">{this.renderContent()}</div>
+          <div class="absolute inset-0">
+            <GeneratedAnswerPlaceholder />
+          </div>
+          <div class="typing-loader-wrapper absolute inset-x-0 bottom-0 w-full z-1 pt-40">
+            <TypingLoader />
+          </div>
+        </div>
+      );
+    } else {
+      return this.renderContent();
+    }
+  }
```

Comment:
Could we also add a check in this function for `this.isAnswerVisible` ? In my opinion, the placeholders should not show when toggling off.

https://github.com/coveo/ui-kit/assets/78121423/8219d0c2-a005-43d2-8384-bad991328b29

Reply:
@alexprudhomme please takeover this PR and do this :)

Reply:
this has been added